lib/fs/basicfs.go

```diff
@@ -105,6 +317,14 @@ func (f fsFile) Stat() (FileInfo, error) {
 	return fsFileInfo{info}, nil
 }
 
+func (f fsFile) Sync() error {
+	err := f.File.Sync()
+	if err != nil && !strings.Contains(err.Error(), "handle is invalid") {
```

Comment:
On windows fsyncing directories fails with this error. I should probably add an OS check and a comment to make that easier to understand.